che-theia-plugins.yaml

```diff
@@ -371,11 +357,6 @@ plugins:
     extensions:
       - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-kubernetes-tools/vscode-kubernetes-tools-1.2.1.vsix
       - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-yaml/vscode-yaml-0.8.0.vsix
-  - repository:
-      url: https://github.com/rogalmic/vscode-bash-debug
-      revision: v0.3.9
-    extensions:
-      - https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix
```

Comment:
https://github.com/rogalmic/vscode-bash-debug/releases/download/untagged-78eaa7784342409ec2a3/bash-debug-0.3.9.vsix ?

Reply:
I'll re-introduce it in a separate PR

Reply:
Done :+1: